client/my-sites/site-settings/jetpack-credentials/index.jsx

```diff
@@ -25,13 +26,28 @@ import RewindCredentialsForm from 'components/rewind-credentials-form';
 import './style.scss';
 
 class JetpackCredentials extends Component {
+	isSectionHighlighted() {
+		if ( ! window.location.hash ) {
+			return null;
```

Comment:
This should return false instead of null. 
This way the function will always return a boolean. (vs true and null)
```suggestion
			return false;
```